src/main/java/omeis/providers/re/HSBStrategy.java

```diff
@@ -329,7 +340,6 @@ private void render(RGBBuffer buf, PlaneDef planeDef) throws IOException,
         performanceStats.startRendering();
         int n = tasks.length;
         Future[] rndTskFutures = new Future[n]; // [0] unused.
-        ExecutorService processor = Executors.newCachedThreadPool();
 
         while (0 < --n) {
```

Comment:
Since we're possibly sharing the thread pool now I think the below needs to be refactored slightly to take that into account. In particular the `shutdown()` needs to be removed, no?